### DISCUSSION TITLE
Far-Sighted and Near-Sighted trait should not be mutually exclusive

### DISCUSSION THREAD

**Discussion by AngelicosPhosphoros** [cataclysmbn/Cataclysm-BN#2074] [General]
### Describe the bug

When starting game, player cannot choose both near sighted and far-sighted traits.

### Steps To Reproduce

1. Start to choosing player traits
2. Choose near-sighted
3. Now you cannot choose far-sighted

### Expected behavior

Near-Sighted people became far-sighted to 50 years old and then keep to have both eye disorders.
The game have even an item for such people: bifocal glasses.

So this traits must be choosable both, and player should start with bifocal glasses.

### Versions and configuration

Version `de64c76`.

**Comment by AngelicosPhosphoros** [comment]
Probably caused by mutations code:
https://github.com/cataclysmbnteam/Cataclysm-BN/blob/de64c76478c5f437fc8af0971962875000431a21/data/json/mutations/mutations.json#L764
https://github.com/cataclysmbnteam/Cataclysm-BN/blob/de64c76478c5f437fc8af0971962875000431a21/data/json/mutations/mutations.json#L718

**Comment by Coolthulhu** [comment]
The reason they are exclusive is balance: contacts, bifocal glasses, and eye replacing CBM affect both traits, so the player was incentivized to take either both of the traits or none.

**Comment by AngelicosPhosphoros** [comment]
We could split contacts into 2 groups (and make player cannot actually wear both kinds); bifocal glasses seem to spawn rarer other kinds and for CBMs it seems to be expected.
